cpu/stm32/stmclk/stmclk_l4wx.c

```diff
@@ -339,7 +339,9 @@
 #define CLOCK48MHZ_USE_MSI          0
 #endif
 
-#if IS_ACTIVE(CLOCK48MHZ_USE_PLLQ)
+#if defined(CPU_FAM_STM32WL)
```

Comment:
Instead of adding a specific case for stm32wl here, I would just modify line 335 as follows:
```
#elif CONFIG_CLOCK_MSI == MHZ(48) && !defined(CPU_FAM_STM32WL)
```

Then both CLOCK48MHZ_USE_PLLQ and CLOCK48MHZ_USE_MSI will be 0 (according to the manual there's no specific 48MHz clock on stm32wl).